samples/client/petstore-security-test/typescript-inversify/IHttpClient.ts

```diff
@@ -0,0 +1,12 @@
+import * as Rx from "rx";
+import HttpResponse from "./HttpResponse";
+import { Dictionary } from "lodash";
+
+interface IHttpClient {
+    get(url:string, headers?:Dictionary<string>):Rx.Observable<HttpResponse>
```

Comment:
do you really want to use a lodash dictionary? why not use a plain JavaScript object?

Reply:
The lodash dictionary restrict the type of values and not permit nested Object. Also Angular's HttpClient allows only string as value ([Header]: https://angular.io/api/http/Headers)

Reply:
you can restrict the value of an object as follows:
```
let myObject: {[key: string]: string}
```

Reply:
You have reason, i'm removing lodash.